### ISSUE TITLE
Assessment Results

### ISSUE THREAD

**Issue by francepack** [saraho1123/mod_0_skills#1]
@saraho1123 
You've performed well enough to merit advancing with some additional revisions to your assessment. Well done! Please have revisions in by Monday. Here's what we'd like to see updated:

- Array attributes typically have plural names. How could you rephrase `workTrainedFor` to be more simple, and plural?

- Your boolean attributes could each use a word added to the front to help communicate the question that the variable answers. This is a common pattern you'll see. `isDelicious` from your variables file is an example of this pattern.

Let me know if you have questions!

**Comment by saraho1123** [comment]
@francepack 
Thanks so much Mason! I updated the array and booleans. 

I have a question though. I made my updates very simple. I am wondering if it would have been ok if I had put this for Class 1 method boolean:

  `isClockedIn` (I updated it to isWorking)

I think specifically what I am asking is if the boolean method needs to have the same word/phrase as the boolean attribute? Would it have been acceptable to just put the `is` in front of `clockedIn` ?

Hopefully that question makes sense. Thanks Mason!

**Comment by saraho1123** [comment]
@francepack 
So I looked at it a little more and realized that you said it was the attribute booleans, not the method booleans that needed revision. I have made additional edits. I remember being nervous about my boolean methods, and so I just assumed that's what was wrong. Bad to assume!

I think this also answers my question about naming boolean methods. But definitely let me know if I'm off track at all!
